sdks/go/test/integration/transforms/xlang/inference/inference.go

```diff
@@ -0,0 +1,50 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inference
+
+import (
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	_ "github.com/apache/beam/sdks/v2/go/pkg/beam/runners/dataflow"
+	_ "github.com/apache/beam/sdks/v2/go/pkg/beam/runners/flink"
+	_ "github.com/apache/beam/sdks/v2/go/pkg/beam/runners/universal"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/testing/passert"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/transforms/xlang/inference"
+)
+
+func SklearnInference(expansionAddr string) *beam.Pipeline {
+	p, s := beam.NewPipelineWithRoot()
+
+	beam.Impulse(s)
+
+	inputRow := [][]int64{{0, 0}, {1, 1}}
+	input := beam.CreateList(s, inputRow)
+	kwargs := inference.SklearnKwargs{
+		ModelURI: "/tmp/staged/sklearn_model",
```

Comment:
Where/how is this staged? where is this file?
Is it automatically done from the python code?

Reply:
I was expecting it to be staged by python code and then we could have used the `semi_persist_dir` to load the model from there. Seems like java uses `semi_persist_dir`. For the test, I manually uploaded the model file to gcs bucket and passed the `gs://...` address there. Updated the PR description with job link.  

Tagging @chamikaramj to know how java loads the model. If it is loaded from `semi_persist_dir` do we have to change something on Go SDK side or is it staged by the python code?